plugins/crane.yaml

```diff
@@ -0,0 +1,130 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: crane
+spec:
+  homepage: https://github.com/gocrane/kubectl-crane
+  shortDescription: "Easily interact with Crane"
+  description: |
+    Crane is a FinOps Platform for Cloud Resource Analytics 
+    and Economics in Kubernetes clusters.
+    Easily manage the recommendation resource with the plugin.
+  caveats: |
+    For additional options:
```

Comment:
Typically it's not useful to print usage strings in plugin caveats or description. They go out of date fairly quickly and half of the commands here aren’t relevant to users. What do you think about explaining the functionality using some sentences in "description" field?